### DISCUSSION TITLE
I'm unable to login to bitwarden on chrome, firefox works OK

### DISCUSSION THREAD

**Discussion by wptracy** [dani-garcia/vaultwarden#3219] [General]
I'm unable to login to bitwarden extension on chrome, firefox works OK

I've configured the Server URL SELF-HOSTED ENVIRONMENT to https://10.0.0.233:7277

I can login on firefox, I can't login on chrome.

Error: Failed to fetch.

There are no firewalls.  Windows fails and Ubuntu linux fails .

![Screenshot from 2023-02-02 09-58-00](https://user-images.githubusercontent.com/78648145/216391572-34ca95af-6e60-42c5-9237-b43776b5459e.png)

**Comment by wptracy** [comment]
I went to https://10.0.0.233:7277/admin

and then tried to login to bitwarden and it worked.

**Reply by BlackDex** [reply]
The admin interface is different. 
Make sure you have the latest version of both Vaultwarden and the clients like the extensions

**Reply by wptracy** [reply]
My home assistant add-on is Vaultwarden (Bitwarden)
Current version: 0.19.0 

The firefox extension is:
 Bitwarden
© Bitwarden Inc. 2015-2023
Version: 2023.1.0
Server version (Third-party): 1.27.0 

The chrome extension is:
Bitwarden
© Bitwarden Inc. 2015-2023
Version: 2023.1.0
Server version (Third-party): 1.27.0

I wasn't able to login to chrome until I https://10.0.0.233:7277/admin

I have no idea why it works now.

I closed all windows and tried again to login on chrome extension and it still works.

I hope it stays working.  I spent 5 hours googling and messing with this.

**Reply by BlackDex** [reply]
No clue either. Maybe something strange from Home Assistant.
